server/grpcserver.go

```diff
@@ -2,13 +2,21 @@ package server
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/babylonlabs-io/finality-gadget/proto"
 	"github.com/babylonlabs-io/finality-gadget/types"
+	"go.uber.org/zap"
 )
 
 // QueryIsBlockBabylonFinalized is an RPC method that returns the finality status of a block by querying the internal db.
 func (s *Server) QueryIsBlockBabylonFinalized(ctx context.Context, req *proto.QueryIsBlockBabylonFinalizedRequest) (*proto.QueryIsBlockFinalizedResponse, error) {
+	s.logger.Info(
```

Comment:
imo we should change these to debug logs, these grpc calls happen quite often so can crowd out other logs if always enabled

Reply:
Agreed, and the message can be a bit more informative, e.g., `QueryIsBlockBabylonFinalized` -> `QueryIsBlockBabylonFinalized is called`